msprime/mutations.py

```diff
@@ -187,6 +209,466 @@ def __init__(
         super().__init__(alleles, root_distribution, transition_matrix)
 
 
+class BLOSUM62(MutationModel):
+    def __init__(self):
+        alleles = _AMINO_ACIDS
+        num_alleles = len(alleles)
```

Comment:
I would be good to document where these values came from - can you give some notes here on where you got them? Some of the values are at 3 digits of precision, others at 12. Good to give some provenance on that also.

Reply:
True. These are the values that are used in both pyvolve and seqgen for their rate matrices. I still need to check whether there is a better way, meaning deriving the ratet matrix from the time-dependent probability matrices as originally published. And whether it makes sense to actually do this each time, or if the best way is to store the raw values as is done in pyvolve and seq-gen.

Reply:
Copying in the values like this is totally fine - just a note on where they came from is all we need.